teaspoons-cli/teaspoons/config.py

```diff
@@ -8,15 +8,36 @@
 class CliConfig:
     """A class to hold configuration information for the CLI"""
 
+    # def __init__(self, env_file: str = "../.env"):
+    #     self.config = dotenv_values(env_file)
+
+    #     if 'tests' not in env_file:
+    #         self.client_info = OAuth2ClientInfo.from_oidc_endpoint(
+    #             self.config["OAUTH_OPENID_CONFIGURATION_URI"],
+    #             client_id=self.config["OAUTH_CLIENT_ID"],
+    #             scopes=[f"openid+email+profile+{self.config['OAUTH_CLIENT_ID']}"]
+    #         )
+    #     else:
+    #         self.client_info = None
+
     def __init__(self):
         self.config = dotenv_values("../.env")
```

Comment:
Depending on how this is likely to be used, you could maybe have a config file in the home directory, like `~/.teaspoons-config` or something. Although I'm not sure if that would make the most sense for your use case.

Reply:
i do like calling the file `.teaspoons-config` rather than the generic `.env`... i don't really know how to think about where it should live. currently we're providing the file in the repo, but it's a future task to figure out whether that's the best way to do it or if it should be generated or pulled from somewhere else. given that users are ultimately going to pull this package from PyPi, do you have advice on how we should provide these values?